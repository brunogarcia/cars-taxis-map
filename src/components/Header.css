.Header-main a {
  color: #333;
}
