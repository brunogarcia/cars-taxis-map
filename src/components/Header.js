import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import AppBar from '@material-ui/core/AppBar';
import Toolbar from '@material-ui/core/Toolbar';
import Typography from '@material-ui/core/Typography';

const styles = {
  root: {
    flexGrow: 1,
  },
};

function Header() {
  return (
    <div>
      <AppBar position="static" color="default">
        <Toolbar>
          <Typography variant="title" color="inherit">
            Cars &amp; Taxis
          </Typography>
        </Toolbar>
      </AppBar>
    </div>
  );
}

export default withStyles(styles)(Header);
