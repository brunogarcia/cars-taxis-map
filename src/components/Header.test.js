import React from 'react';
import renderer from 'react-test-renderer';
import Header from './Header';

test('renders correctly', () => {
  const component = renderer.create(<Header />);

  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
