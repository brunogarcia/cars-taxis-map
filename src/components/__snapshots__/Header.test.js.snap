// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <header
    className="MuiPaper-root-10 MuiPaper-elevation4-16 MuiAppBar-root-2 MuiAppBar-positionStatic-6 MuiAppBar-colorDefault-7 Header-main"
  >
    <div
      className="MuiToolbar-root-37 MuiToolbar-gutters-38"
    >
      <h3
        className="MuiTypography-root-39 MuiTypography-subheading-46"
      >
        A Front-end test by 
        <a
          href="http://blog.garciaechegaray.com/about-me/"
        >
          Bruno Garcia
        </a>
      </h3>
    </div>
  </header>
</div>
`;
